Cargo.toml

```diff
@@ -11,9 +11,9 @@ crate-type = ["cdylib"]
 [dependencies]
 pyo3 = "0.19.0"
 anyhow = "1.0"
-cedar-policy = "~2.2.0"
-cedar-policy-cli = "~2.2.0"
-cedar-policy-formatter = "~2.2.0"
+cedar-policy = "~4.1.0"
```

Comment:
There is a 4.2.2 release available on https://docs.rs/cedar-policy/latest/cedar_policy/ however the public docs https://docs.cedarpolicy.com/ currently refers to 4.1 hence this choice

Reply:
Thanks for thinking about the versioning.

I think we might jump to a 4.x version number to communicate support for Cedar 4.x

It looks like CedarJavaFFI did that:
https://github.com/cedar-policy/cedar-java/blob/main/CedarJavaFFI/Cargo.toml#L9

Not asking you to do anything here. Just FYI..